wire-library/wire-java-generator/build.gradle.kts

```diff
@@ -24,3 +29,9 @@ dependencies {
   testImplementation(deps.assertj)
   testImplementation(deps.jimfs)
 }
+
+configure<MavenPublishBaseExtension> {
```

Comment:
Can maybe move this to the top level build file later? Can check which Kotlin plugin the subproject applies to pick the right `Kotlin*` config.

Reply:
Yeah, I don’t mind this that much!